Makefile

```diff
@@ -2,23 +2,6 @@ VERSION := $(shell git describe --dirty=-dirty --always)
 
```

Comment:
I notice you use numbered releases for this project. Do you want to add `--tags` to the `VERSION` to include that release number?